### ISSUE TITLE
Dynamic markers fail to render past first

### ISSUE THREAD

**Issue by LukevanTricht** [Vheissu/aurelia-google-maps#62]
When adding markers through binding the markers array, aurelia-google-maps renders the first marker, but fails to render any additional markers beyond that.

**Comment by ghost** [comment]
Post some sample code? :)

**Comment by LukevanTricht** [comment]
Apologies for forgetting to add some sample code.

HTML Part of Template:
``` html
<template>
  <google-map map-type="ROADMAP" map-click.delegate="mapClickHandler($event)" markers.two-way="officeLocations" auto-update-bounds="true"></google-map>
</template>
```

Typescript Object:
``` Typescript
import {bindable} from 'aurelia-framework'
export class OrganizationMap{
  @bindable officeLocations:IMarker[] = [];
  @bindable selectedLocation:ILocation;
  constructor(){}
  mapClickHandler(event) {
    let latLng = event.detail.latLng,
      lat = latLng.lat(),
      lng = latLng.lng();
    this.selectedLocation = {lat: lat, long: lng};
  }
}
export interface ILocation{
  lat:string
  long:string
  address?:string
}
export interface IMarker{
  latitude:number
  longitude:number
  icon?:string
  title?:string
}
```

I've developed a fix for this issue in Pull Request #63
